google-cloud-bigtable/scripts/regenerate.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
```

Comment:
I am hesitant to have these bigtable-specific scripts checked into google-cloud-java. My main concern is them becoming abandoned over time and becoming deadweight because nobody knows how to use them. I'm ok with having them checked in here if we commit to generalizing them for other API owners (which would be an awesome result); otherwise, I would prefer to keep them outside of the repo.

Reply:
I found them very useful when when testing gapic config changes against the client. I can work to generalize them, but I think it would require a sync on your plans for autogenerating clients in general.
Can we compromise on leaving them here for now and I will  circle back to them?

Reply:
Yes, I just want to make sure we circle back :-)